webapi/src/main/resources/knora-ontologies/knora-base.ttl

```diff
@@ -2001,6 +2010,37 @@
 ###  http://www.knora.org/ontology/knora-base#StillImageFileValue
 
 :StillImageFileValue
+    rdf:type        owl:Class ;
+    rdfs:subClassOf :StillImageAbstractFileValue,
+                    [ rdf:type        owl:Restriction ;
+                      owl:onProperty  :dimY ;
+                      owl:cardinality "1"^^xsd:nonNegativeInteger ],
+                    [ rdf:type        owl:Restriction ;
+                      owl:onProperty  :dimX ;
+                      owl:cardinality "1"^^xsd:nonNegativeInteger ] ;
+    rdfs:comment    "A file containing a two-dimensional still image"@en .
+
+
+###  http://www.knora.org/ontology/knora-base#StillImageExternalFileValue
+
+:StillImageExternalFileValue
+    rdf:type        owl:Class ;
+    rdfs:subClassOf :StillImageAbstractFileValue,
+                    [ rdf:type        owl:Restriction ;
+                      owl:onProperty  :externalUrl ;
+                      owl:cardinality "1"^^xsd:nonNegativeInteger ],
+                    [ rdf:type        owl:Restriction ;
+                      owl:onProperty  :dimY ;
+                      owl:cardinality "1"^^xsd:nonNegativeInteger ],
+                    [ rdf:type        owl:Restriction ;
+                      owl:onProperty  :dimX ;
+                      owl:cardinality "1"^^xsd:nonNegativeInteger ] ;
```

Comment:
**suggestion (non-blocking):** It doesn't hut to have it, but since `dimX` and `dimY` are declared on `StillImageAbstractFileValue` I don't think it makes too much sense to re-declare them on both subclasses.

Reply:
=> https://github.com/dasch-swiss/dsp-api/pull/3131/commits/d978fd8fa72b38ea68e256684fefa3c6e6575aec